src/test/java/lotto/LottoRankTest.java

```diff
@@ -0,0 +1,46 @@
+package lotto;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+import static lotto.LottoRank.*;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class LottoRankTest {
+    
+    @Test
+    void 일치_개수로_등수를_찾을_수_있다() {
+        assertThat(LottoRank.fromMatchCount(6)).isEqualTo(FIRST);
+        assertThat(LottoRank.fromMatchCount(5)).isEqualTo(LottoRank.SECOND);
+        assertThat(LottoRank.fromMatchCount(4)).isEqualTo(LottoRank.THIRD);
+        assertThat(LottoRank.fromMatchCount(3)).isEqualTo(LottoRank.FOURTH);
+        assertThat(LottoRank.fromMatchCount(2)).isEqualTo(null);
+    }
+    
+    @Test
+    void 당첨_통계에서_각_등수별_당첨_개수를_셀_수_있다() {
+        Lotto winningLotto = new Lotto(List.of(1, 2, 3, 4, 5, 6));
```

Comment:
```suggestion
        Lotto winningLotto = new Lotto(1, 2, 3, 4, 5, 6);
```
가변 인자를 가지는 생성자를 추가하면 위와 같이 더 편리하게 구현할 수 있지 않을까?
다수의 부 생성자를 추가해 개발자의 편리성을 제공해 본다.